### PR TITLE
Display `inst/AUTHORS` info in authors.html

### LINKED ISSUE #2332: Add a link to `inst/AUTHORS` page if `inst/AUTHORS` exists
If `inst/AUTHORS` exists, CRAN seems to automatically generate a link to that file.

image
https://CRAN.R-project.org/package=prqlr

It would be great if pkgdown could also link to the AUTHORS file in the same way.

For now, the following text is displayed, so it is not clear what `inst/AUTHORS` is to the viewer of this page.

image
https://eitsupi.github.io/prqlr/authors.html

### PR DESCRIPTION
**Pull request by hadley**
Fixes #2332

To do:
* [x] Add test
* [x] News bullet

### CODE REVIEW

File: R/build-home-authors.R [hunk 1]
```diff
@@ -2,6 +2,13 @@ data_authors <- function(pkg = ".", roles = default_roles()) {
   pkg <- as_pkgdown(pkg)
   author_info <- pkg$meta$authors %||% list()
 
+  inst_path <- path(pkg$src_path, "inst", "AUTHORS")
+  if (length(inst_path) == 1 && file_exists(inst_path)) {
```
Comment (maelle) [inline R/build-home-authors.R:6]: why would the length not be 1?

File: inst/BS5/templates/content-citation-authors.html [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
     </div>
 
-    <div class="section level2 citation">
+    <div class="section level2">
       <h2>{{#translate}}{{authors}}{{/translate}}</h2>
       {{{before}}}
       <ul class="list-unstyled">
         {{#authors}}
-        <li>
+        <µli>
```
Comment (maelle) [inline inst/BS5/templates/content-citation-authors.html:13]:
```suggestion
        <li>
```

File: tests/testthat/test-build-home-citation.R [hunk 3]
```diff
@@ -29,6 +29,15 @@ test_that("source link is added to citation page", {
   expect_true(any(grepl("<code>inst/CITATION</code></a></small>", lines)))
 })
 
+test_that("citation page includes inst/AUTHORS", {
+  pkg <- local_pkgdown_site(test_path("assets/inst-authors"))
+  suppressMessages(init_site(pkg))
```
Comment (maelle) [inline tests/testthat/test-build-home-citation.R:34]: #2330 will be nice :sweat_smile:

File: tests/testthat/test-build-home-authors.R [hunk 4]
```diff
@@ -37,15 +37,9 @@ test_that("Data authors can accept different filtering", {
...
-  pkg <- as_pkgdown(test_path("assets/sidebar-comment"))
-  expect_null(data_authors_page(pkg)$after)
-  expect_null(data_authors_page(pkg)$before)
-
-  pkg$meta$authors$before <- "Dream team:"
-  pkg$meta$authors$after <- "You are welcome!"
-  expect_equal(data_authors_page(pkg)$before, "<p>Dream team:</p>")
-  expect_equal(data_authors_page(pkg)$after, "<p>You are welcome!</p>")
+test_that("authors data includes inst/AUTHORS", {
+  pkg <- as_pkgdown(test_path("assets/inst-authors"))
```
Comment (maelle) [inline tests/testthat/test-build-home-authors.R:41]: would the test be easier to read if the test package were created here, with some test helper, and if the inst/authors file was then added inline in this test?

I thought there was an issue about test assets but I can't find it.

Comment (maelle) [reply tests/testthat/test-build-home-authors.R:41]: Maybe as part of https://github.com/r-lib/pkgdown/issues/2509

File: R/build-home-authors.R [hunk 5]
```diff
@@ -2,6 +2,13 @@ data_authors <- function(pkg = ".", roles = default_roles()) {
...
+    inst <- read_lines(inst_path)
```
Comment (maelle) [inline R/build-home-authors.R:7]: does it need to be Markdownified? probably not?

File: R/build-home-authors.R (hunk 1 above)
Comment (hadley) [reply R/build-home-authors.R:6]: Hmmmm, good question.

File: R/build-home-authors.R (hunk 5 above)
Comment (hadley) [reply R/build-home-authors.R:7]: I don't think we can assume this file is markdown, so the best we can do is put in a `<pre>` tag.

File: tests/testthat/test-build-home-authors.R (hunk 4 above)
Comment (hadley) [reply tests/testthat/test-build-home-authors.R:41]: Yeah, I think that would probably be the case for many of the tests, but requires quite a bit of test infrastructure investment. We'll see how I go.

File: tests/testthat/test-build-home-citation.R (hunk 3 above)
Comment (hadley) [reply tests/testthat/test-build-home-citation.R:34]: Btw I'm not convinced it will be, because I think there are two meanings of `quiet` currently.

**Pull request merged**